core/shared/src/main/scala/scodec/bits/ByteVector.scala

```diff
@@ -1218,6 +1233,13 @@ object ByteVector {
         i += 1
       }
     }
+    def copyToBuffer(buffer: ByteBuffer, offset: Long, size: Int): Unit = {
+      var i = 0
+      while (i < size && buffer.remaining > 0) {
```

Comment:
This implementation is only used for the anonymous `At` subtypes that only define an `apply(idx: Long): Byte` method -- it's overridden in `AtArray` and `AtByteBuffer`. Ideally, I'd like to get rid of the anonymous subtypes and define an `AtFunction`, but I figured that's a bigger refactoring.